indico/modules/events/contributions/templates/forms/contribution_person_widget.html

```diff
@@ -82,8 +80,8 @@
 
             function renderPerson(person) {
                 var $personRow = $('<div>').addClass('person-row');
-                var $personName = $('<span>').text(person.name);
-                var $personStatus = $('<span>').addClass('person-status');
+                var $personName = $('<div>').addClass('name').text(person.name);
+                var $personStatus = $('<div>').addClass('person-status');
                 var $personRoles = $('<span>').addClass('person-roles');
                 var $personButtons = $('<span>').addClass('person-buttons');
                 var $buttonRemove = $('<a>').addClass('i-button-icon danger icon-close').attr('title', $T.gettext("Remove person"));
```

Comment:
I think we all have made our points very clearly for the 3rd or 4th time. Since there's no clear best, let's keep it to personal preference instead bikeshedding on every PR.